specifyweb/frontend/js_src/lib/components/FormPlugins/DateInput.tsx

```diff
@@ -140,6 +139,7 @@ export function DateInput({
           (precision === 'month-year' && !monthSupported)) ? (
           <Button.Icon
             aria-label={formsText.today()}
+            className="print:hidden"
```

Comment:
`print:hidden` should have been here all along - unrelated to the other changes in this PR